src/pcapi/routes/native/v1/account.py

```diff
@@ -0,0 +1,29 @@
+from flask_jwt_extended import get_jwt_identity
+from flask_jwt_extended import jwt_required
+
+from pcapi.core.users.models import VOID_FIRST_NAME
+from pcapi.models.api_errors import ApiErrors
+from pcapi.models.user_sql_entity import UserSQLEntity
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+from .serialization import account
+
+
+@blueprint.native_v1.route("/me", methods=["GET"])
+@spectree_serialize(
+    response_model=account.UserProfileResponse,
+    on_success_status=200,
+    api=blueprint.api,
+)  # type: ignore
+@jwt_required
+def get_user_profile() -> account.UserProfileResponse:
+    identifier = get_jwt_identity()
+    user = UserSQLEntity.query.filter_by(email=identifier).first()
+
+    if user is None:
+        raise ApiErrors({"email": ["Utilisateur introuvable"]})
+
+    return account.UserProfileResponse(
+        first_name=user.firstName if user.firstName != VOID_FIRST_NAME else None, email=user.email
```

Comment:
```suggestion
        first_name=user.firstName or None, email=user.email
```
Je pense que l'on peut se passer du `VOID_FIRST_NAME`

Reply:
Je trouve ça pas mal d'avoir cette constante, on peut l'ajouter dans ton ticket de création de compte, comme ça on voit directement que ça fait référence à ce remplissage par défaut